ftpvl/fetchers.py

```diff
@@ -68,16 +74,88 @@ def __init__(
         project: str,
         jobset: str,
         eval_num: int = 0,
+        absolute_eval_num: bool = False,
         mapping: dict = None,
         hydra_clock_names: list = None
     ) -> None:
         super().__init__()
         self.project = project
         self.jobset = jobset
         self.eval_num = eval_num
+        self.absolute_eval_num = absolute_eval_num
         self.mapping = mapping
         self.hydra_clock_names = hydra_clock_names
 
+    def _get_builds(self, eval_num: int, absolute_eval_num: bool, params: str = "") -> List[int]:
+        """
+        Recursive function that returns a list of build numbers given an eval_num
+        and whether it is an absolute eval num.
+
+        Parameters
+        ----------
+        eval_num : int
+            An integer that specifies the evaluation to download. Functionality
+            differs depending on whether `absolute_eval_num` is True
+        absolute_eval_num : bool
+            Flag that specifies if the eval_num is an absolute identifier instead of
+            a relative identifier. If True, the fetcher will find an evaluation
+            with the exact ID in `eval_num`. If False, `eval_num` should be a
+            non-negative integer with `0` being the latest evaluation and `1` being
+            the second latest evaluation, etc.
+        params : str
+            A string of query parameters used when fetching the evaluations.
+            Most commonly used for pagination. By default, "".
+
+        Returns
+        -------
+        List[int]
+            A list of builds that correspond with the eval_num
+
+        Raises
+        ------
+        ConnectionError
+            Raised if the HTTP request to get the evaluations fails.
+        IndexError
+            Raised if the relative eval_num is not valid.
+        ValueError
+            Raised if the eval_num has no associated builds.
+        """
+        resp = requests.get(
+            f"https://hydra.vtr.tools/jobset/{self.project}/{self.jobset}/evals{params}",
+            headers={"Content-Type": "application/json"},
+        )
+        if resp.status_code != 200:
+            raise ConnectionError("Unable to get evals from server.")
+        evals_json = resp.json()
+        
+        if not self.absolute_eval_num: # if relative eval_num
+            if eval_num >= len(evals_json["evals"]):
+                # check if there is a second page
+                if "next" in evals_json:
+                    return self._get_builds(
+                        eval_num - len(evals_json["evals"]),
+                        absolute_eval_num,
+                        evals_json["next"] # query param for next page
+                    )
+                else:
+                    raise IndexError(f"Invalid eval_num: {self.eval_num}")
```

Comment:
It would be better for this error to mirror the other case below, namely:
```suggestion
                    raise ValueError(f"Unable to find relative eval_num {self.eval_num}")
```
This is the one case where `self.eval_num` is not necessarily the same as the local `eval_num`, and it's definitely the former value we need here, so all's good on that front.